test/tests/bootstrap/test_redfish10_os_bootstrap_base.py

```diff
@@ -42,26 +75,39 @@ def wait_for_task_complete(taskid, retries=60):
     print "Task failed with the following state: " + result['json']['TaskState']
     return False
 
+# download RackHD config from host
+rackhdconfig = fit_common.rackhdapi('/api/2.0/config')['json']
+httpProxies = rackhdconfig['httpProxies']
+rackhdhost = "http://" + str(rackhdconfig['apiServerAddress']) + ":" + str(rackhdconfig['apiServerPort'])
+
+# helper routine for selecting OS image path by matching proxy path
+def proxySelect(tag):
+    for entry in httpProxies:
+        if tag in entry['localPath']:
+            return entry['localPath']
+    return ''
+
 # ------------------------ Tests -------------------------------------
 from nose.plugins.attrib import attr
-@attr(all=True, regression=True)
+@attr(all=True)
 class os_bootstrap_base(fit_common.unittest.TestCase):
     def setUp(self):
         #delete active workflows for specified node
         fit_common.cancel_active_workflows(NODE)
-    #@fit_common.unittest.skipUnless('esxi55' in fit_common.GLOBAL_CONFIG['repos']['os'], "Skipping ESXi5.5")
-    @fit_common.unittest.skipUnless(False, "Skipping ESXi5.5")
+
+    @fit_common.unittest.skipUnless(proxySelect('ESXi/5.5') != '', "Skipping ESXi5.5")
```

Comment:
It might be helpful to add something like "no image repo configured", to the "Skipping" message.  Then they don't have to look at the code to figure out why the test was skipped.